config/dispatcher/dispatcher.ex

```diff
@@ -1,532 +1,566 @@
+
+  match "/fractietypes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/fractietypes/"
   end
-  match "/geboortes/*path" do
+
+  match "/geboortes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/geboortes/"
   end
-  match "/lijsttypes/*path" do
+
+  match "/lijsttypes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/lijsttypes/"
   end
-  match "/kandidatenlijsten/*path" do
+
+  match "/kandidatenlijsten/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/kandidatenlijsten/"
   end
-  match "/lidmaatschappen/*path" do
+
+  match "/lidmaatschappen/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/lidmaatschappen/"
   end
-  match "/mandaten/*path" do
+
+  match "/mandaten/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/mandaten/"
   end
-  match "/bestuursfunctie-codes/*path" do
+
+  match "/bestuursfunctie-codes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/bestuursfunctie-codes/"
   end
-  delete "/mandatarissen/:id" do
+
+  delete "/mandatarissen/:id", %{ layer: :api_services, accept: %{ any: true } } do
     Proxy.forward conn, [], "http://mandataris-archive/" <> id <> "/archive"
   end
-  match "/mandatarissen/*path" do
+  
+  match "/mandatarissen/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/mandatarissen/"
   end
-  match "/mandataris-status-codes/*path" do
+
+  match "/mandataris-status-codes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/mandataris-status-codes/"
   end
-  match "/beleidsdomein-codes/*path" do
+
+  match "/beleidsdomein-codes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/beleidsdomein-codes/"
   end
-  match "/personen/*path" do
+
+  match "/personen/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/personen/"
   end
-  match "/geslacht-codes/*path" do
+
+  match "/geslacht-codes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/geslacht-codes/"
   end
-  match "/identificatoren/*path" do
+  
+  match "/identificatoren/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/identificatoren/"
   end
 
-  match "/tijdsintervallen/*path" do
+  match "/tijdsintervallen/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/tijdsintervallen/"
   end
 
-  match "/mock/sessions/*path" do
+  match "/mock/sessions/*path", %{ layer: :api_services, accept: %{ any: true } } do
```

Comment:
Good catch. Double checked. The provided content-type is `application/vnd.api+json`. But the accept is indeed `*/*`